tests/helpers/mobiledoc.js

```diff
@@ -0,0 +1,26 @@
+import PostNodeBuilder from 'content-kit-editor/models/post-node-builder';
+import MobiledocRenderer from 'content-kit-editor/renderers/mobiledoc';
+
+/*
+ * makeMD({post, section, marker, markup} =>
+     post([
+       section('P', [
+         marker('some text', [markup('B')])
+       ])
+     })
+ */
+function makeMD(treeFn) {
```

Comment:
Maybe change it to `build` or just `make`? It is used by `Helpers.mobiledoc` as `Helpers.mobiledoc.makeMD`, already slightly redundant. Changing to `makeMobiledoc` would make the code read `Helpers.mobiledoc.makeMobiledoc`.

The mobiledoc should be valid — it first builds a post and then converts it to mobiledoc.